src/main/java/com/cs3250p1/project1/BarChartEx.java

```diff
@@ -1,77 +1,95 @@
 package com.cs3250p1.project1;
```

Comment:
Alright, nice to get the final version of this bar graph, looks great and fulfills the user story